setup.cfg

```diff
@@ -99,14 +99,18 @@ extras = True
 [tool:pytest]
 # Options for py.test:
 # Specify command line options as you would do when invoking py.test directly.
-# e.g. --cov-report html (or xml) for html/xml output or --junitxml junit.xml
-# in order to write a coverage file that can be read by Jenkins.
+# e.g. --junitxml junit.xml in order to write a coverage file that can be read
+# by Jenkins.
 addopts =
-    --cov pyscaffold --cov-config .coveragerc --cov-report term-missing
+    --cov pyscaffold
+    --cov-config .coveragerc
+    --cov-report=
```

Comment:
Is there the `term-missing` parameter missing? Or is this preventing `pytest-cov` to do something wrongly?

Reply:
With `=` followed, i have tried to suppress the incorrect output (http://pytest-cov.readthedocs.io/en/latest/reporting.html)